spec/lib/miq_automation_engine/service_methods/miq_ae_service_ems_event_spec.rb

```diff
@@ -170,11 +170,4 @@
       @service_event.change_event_target_state("src_vm", "suspended")
     end
   end
-
-  %w(disconnect_storage refresh_on_reconfig).each do |method|
```

Comment:
@jameswnl I think `src_vm_disconnect_storage` is still a valid method so this test is still needed